dfastmi/gui/dialog_view.py

```diff
@@ -614,7 +619,34 @@ def _create_branch_input(self, layout: QFormLayout) -> None:
             self._branch.addItem(b.name)
         self._branch.setCurrentText(self._view_model.current_branch.name)
         layout.addRow(gui_text("branch"), self._branch)
+    
+    def _create_case_input(self, layout: QBoxLayout) -> None:
+        """
+        Create input field for case description.
+
+        Args:
+            layout (QBoxLayout): Layout to add the case description input field.            
+
+        Returns:
+            None
+        """
+        self._case_description = QLineEdit(self._win)        
+        self._case_description.setText(self._view_model.model.case_description)
+        self._case_description.editingFinished.connect(self._update_case_description)        
+        self._case_description.setToolTip(gui_text("case_description_tooltip"))
+        case_description_label = QLabel(gui_text("case_description"), self._win)
+        layout.addRow(case_description_label, self._case_description) 
+
+    def _update_case_description(self) -> None:
+        """
+        Update case description.
+
+        Returns:
```

Comment:
FYI, if a function returns None, it is better to not document this.

Reply:
Good to know!

Reply:
Will remove here and keep in mind for future. Will not update rest of code.